### PR TITLE
Add trivy verification phase to the pipeline

### PR DESCRIPTION
**Pull request by fmulero**
Signed-off-by: Fran Mulero <fmulero@vmware.com>

**Description of the change**
Add trivy verification phase to the pipeline

**Benefits**

Run CVE scan during verification phase

### CODE REVIEW

File: .vib/vib-scheduled-verify.json [hunk 1]
```diff
@@ -23,6 +23,23 @@
           "action_id": "container-image-lint"
         }
       ]
+    },
+    "verify": {
+      "context": {
```
Comment (recena) [inline .vib/vib-scheduled-verify.json:28]: Do you need this section here?

**Comment by fmulero** [comment]
I am going to close this PR because the workflow is not triggered, I have to check why

File: .vib/vib-scheduled-verify.json (hunk 1 above)
Comment (fmulero) [reply .vib/vib-scheduled-verify.json:28]: Yes, you are right. I need some additional parameterization

Comment (recena) [reply .vib/vib-scheduled-verify.json:28]: I doubt if we need the `target_platform` since we are not deploying...

Comment (fmulero) [reply .vib/vib-scheduled-verify.json:28]: `target_platform` is not required, Trivy analyse directly the image

**Pull request closed without merging**